File: DataStructureProgramming/JavaCoder/src/stack/stacklinklist/Item.java
package stack.stacklinklist;

public class Item {
    private int num;
    public int data;

    public Item next;
    public Item previous;

    public Item (int id){
        data = id;
        num = id;
        next = null;
    }

    public void displayLink() {
        System.out.println("{ " + data + " }");
    }
}

File: DataStructureProgramming/KotlinCoder/src/stack/stacklinkendlist/Item.kt
package stack.stacklinkendlist

class Item(id: Int) {
    private var num = 0
    var data = 0

    var next: Item? = null
    var previous: Item? = null

    init {
        data = id
        num = id
        next = null
    }

    fun displayLink() {
        println("{ $data }")
    }
}